### ISSUE TITLE
Find new sound for discovering secrets

### ISSUE THREAD

**Issue by serano01** [SuperTux/supertux#1671]
Since not everyone is satisfied with the current one, and the old "Excellent"-sound was unpopular for being not understandable for everyone, I'm looking for a replacement. But I'm open for suggestions, as well.

Two sounds which might fit with the following (when all secrets are revealed) "tada"-sound:

https://opengameart.org/content/win-sound-effect
https://opengameart.org/content/win-sound-1

**Comment by Semphriss** [comment]
Bad tongue moment: The former sounds like it would fit better in a clash-of-clans-like MMORPG mobile game and the latter sounds like someone testing instruments on their electronic piano.

However, I do agree we need to find a replacement for that song, and potentially something too for the secrets-cleared sound effect, if we can find something better than that "tada" sound :)

**Comment by serano01** [comment]
This is the original sound from "Yeti's Revenge", which inspired me to use something with cheering in the first place: https://raw.githubusercontent.com/SuperTux/addons-src/master/niso-yetis-revenge/sounds/clapping.ogg

**Comment by serano01** [comment]
A short blop sound (taken from an earlier version of SuperTux). I kinda like it.
[bump-upgrade.zip](https://github.com/SuperTux/supertux/files/5996595/bump-upgrade.zip)

**Comment by serano01** [comment]
At this point I'm considering using the old "Excellent!"-sound again, as nothing really fitting has been found by me or anyone else.

**Comment by HybridDog** [comment]
What about this sound?
https://freesound.org/people/grunz/sounds/109663/

**Comment by Busybody64** [comment]
We should keep the Excellent sound.

If it ain't broke, don't fix it.

**Comment by serano01** [comment]
> We should keep the Excellent sound.
> 
> If it ain't broke, don't fix it.

Yeah, that seems to be the best solution. I'll ask the team, when they agree I will change it back.

**Comment by Alasdairbugs** [comment]
Closed as sfx was changed
